Sources/PointFree/About.swift

```diff
@@ -22,7 +22,7 @@ private let aboutView = View<RequestContext<Prelude.Unit>> { ctx in
       body(navView.view(ctx) + [
         gridRow([
           gridColumn(sizes: [.xs: 12], [
-            div([`class`([Class.padding.all(4)])], [
+            div([`class`([Class.padding([.mobile: [.all: 4]])])], [
```

Comment:
this is now the interface to adding padding/margin. You specify a dictionary that maps breakpoints to sides+value. If you are only applying a single side it's a lil more verbose, but if you are doing multiple it's shorter:

```
[Class.padding([.mobile: [.leftRight: 2, .topBottom: 3]])]

// vs

[Class.padding(leftRight: 2), Class.padding(topBottom: 3)]
```

and even shorter when you start adding breakpoint specific values.

Reply:
oh also, we can cook up components to automate some stuff. for example, the default padding in a grid column should prob be:

```swift 
let defaultColumnPadding = Class.padding([.mobile: [.all: 3], .desktop: [.all: 4]])
```

And then we can just use `defaultColumnPadding` everywhere.